administrator/components/com_admin/sql/updates/mysql/5.3.0-2024-10-13.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `#__fields` MODIFY `fieldparams` MEDIUMTEXT;
```

Comment:
```suggestion
ALTER TABLE `#__fields` MODIFY `fieldparams` MEDIUMTEXT;

```
Add line feed at end of file.